src/components/Chart/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './styles.css';
+
+// Bootstrap components
+import { Card } from 'reactstrap';
+
+// libraries
+import TradingViewWidget from 'react-tradingview-widget';
+
+class Chart extends Component {
+
+  shouldComponentUpdate = (nextProps) => this.props.selectedCrypto !== nextProps.selectedCrypto
```

Comment:
Is this actually necessary? TradingViewWidget does this too.

Reply:
Removed